packages/shared/src/components/onboarding/OnboardingFeedHeader.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useCallback, useEffect } from 'react';
+import classNames from 'classnames';
+import { useRouter } from 'next/router';
+import { FilterOnboardingV4 } from './FilterOnboardingV4';
+import {
+  Button,
+  ButtonColor,
+  ButtonIconPosition,
+  ButtonSize,
+  ButtonVariant,
+} from '../buttons/Button';
+import { REQUIRED_TAGS_THRESHOLD } from './common';
+import { ArrowIcon } from '../icons';
+import { LayoutHeader } from '../layout/common';
+import { usePrompt } from '../../hooks/usePrompt';
+import { useMyFeed } from '../../hooks/useMyFeed';
+import { useAlertsContext } from '../../contexts/AlertContext';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../../lib/analytics';
+
+const promptConfig = {
+  title: 'Discard tag selection?',
+  description: 'Your personalized feed is only a few tags away',
+  cancelButton: {
+    title: 'Stay',
+    variant: ButtonVariant.Primary,
+    color: ButtonColor.Cabbage,
+  },
+  okButton: {
+    title: 'Leave',
+    variant: ButtonVariant.Secondary,
+  },
+  className: {
+    buttons: 'flex-row-reverse',
+  },
+};
+
+type OnboardingFeedHeaderProps = {
+  isPreviewFeedVisible: boolean;
+  setPreviewFeedVisible: React.Dispatch<React.SetStateAction<boolean>>;
+  isFeedPreviewEnabled: boolean;
+  tagsCount: number;
+};
+
+export const OnboardingFeedHeader = ({
+  isPreviewFeedVisible,
+  setPreviewFeedVisible,
+  isFeedPreviewEnabled,
+  tagsCount,
+}: OnboardingFeedHeaderProps): JSX.Element => {
+  const { updateAlerts } = useAlertsContext();
+  const { trackEvent } = useAnalyticsContext();
+  const { showPrompt } = usePrompt();
+  const router = useRouter();
+  const { registerLocalFilters } = useMyFeed();
+
+  const completeOnboarding = useCallback(() => {
+    registerLocalFilters();
+    updateAlerts({ filter: false, myFeed: 'created' });
+
+    trackEvent({
+      event_name: AnalyticsEvent.CreateFeed,
+    });
+
+    return router.replace({
+      pathname: router.route,
+      query: {
+        welcome: 'true',
```

Comment:
We're removing this in another onboarding experiment, but since this is now added in a different place, not sure if we would want that experiment available here as well.

Reply:
Not sure, maybe product can answer?

Reply:
Will leave as is in this experiment.